docs/build/arm-exception-handling.md

```diff
@@ -139,17 +139,17 @@ The epilogues for canonical functions follow a similar form, but in reverse and
 |-----------------|-----------------------------------|----------|------------|
 |6|*`Stack Adjust`*!=0 and *`EF`*==0|16/32|`add   sp,sp,#xx`|
 |7|*`R`*==1 and *`Reg`*!=7|32|`vpop  {d8-dE}`|
-|8|*`C`*==1 or (*`L`*==1 and *`H`*==0) or *`R`*==0 or *`EF`*==1|16/32|`pop   {registers}`|
-|9a|*`H`*==1 and *`L`*==0|16|`add   sp,sp,#0x10`|
-|9b|*`H`*==1 and *`L`*==1|32|`ldr   pc,[sp],#0x14`|
+|8|*`C`*==1 or (*`L`*==1 and (*`H`*==0 or *`Ret`* !=0)) or *`R`*==0 or *`EF`*==1|16/32|`pop   {registers}`|
+|9a|*`H`*==1 and (*`L`*==0 or *`Ret`*!=0)|16|`add   sp,sp,#0x10`|
+|9b|*`H`*==1 and *`L`*==1 and *`Ret`*==0|32|`ldr   pc,[sp],#0x14`|
```

Comment:
Should we also add an explanation to the Ret==0 case saying that it could be pop {pc} OR it could also be ldr pc,[sp],#0x14?

Reply:
On instruction 8? Maybe, but I'm not sure how to include that information without blowing up the table too much. Because if `R==0` then you'd still have instruction 8 popping a number of registers, but not including `pc`. The logical condition, while complex, should be accurate for when this instruction is present though.